package.json

```diff
@@ -35,7 +35,7 @@
   "scripts": {
     "build:clean": "rm -rf dist",
     "build": "pnpm build:clean && tsup",
-    "publish": "pnpm run build && npm publish",
+    "prepublishOnly": "pnpm run build",
```

Comment:
But this is just build then?  So, does it change anything from the instructions?

Reply:
Oh I see it autoruns.